packages/aws-cdk/lib/cli/telemetry/sanitation-utils.ts

```diff
@@ -0,0 +1,64 @@
+import { CliConfig } from "@aws-cdk/user-input-gen";
+import { Command } from "./schema";
+
+/**
+ * argv is the output of yargs
+ */
+export function sanitizeCommandLineArguments(argv: any, config: CliConfig): Command {
+  const command = argv._[0];
+  const path: string[] = [];
+  const parameters: string[] = [command];
```

Comment:
This is a dictionary according to the RFC? https://github.com/aws/aws-cdk-rfcs/pull/734/files#r2162360512